plugins/headless/hu.bme.mit.gamma.headless.server/src/hu/bme/mit/gamma/headless/server/util/IsDirectoryFilter.java

```diff
@@ -0,0 +1,17 @@
+package hu.bme.mit.gamma.headless.server.util;
+
+import java.io.File;
+import java.io.FileFilter;
+
+public class IsDirectoryFilter implements FileFilter {
```

Comment:
Wouldn't `DirectoryFilter` be a better name?

Reply:
fixed in 82e3b4f